### DISCUSSION TITLE
p:galleria not visible

### DISCUSSION THREAD

**Discussion by kurtomerfaruk** [primefaces/community#3176] [PrimeFaces] [answered]
Hello,
I am using p:galleria in dialog on a page but thumbnails do not appear. Also, when I examine from DevTools, css classes do not appear when I want to add different css and js files. The version I use is 14.0.8. How can I solve such a problem.

PrimeFaces Showcase : ui-galleria-item-wrapper
Project : ui-galleria-items

![Ekran görüntüsü 2024-12-09 202207](https://github.com/user-attachments/assets/77f812d7-f757-4b2f-8cf1-d9e968bbdef1)
![Ekran görüntüsü 2024-12-09 202144](https://github.com/user-attachments/assets/84d4af84-98a4-48f8-bba0-9a8a3834e6a6)

**Comment by kurtomerfaruk** [comment]
I have found the source of the problem, but I have no idea how to solve it. I use Bootstrap tab, when I add it to the active tab in the opening, the problem disappears. However, when I add it to the last tab and switch to this tab, the image is distorted

**Reply by melloware** [reply]
does it work without Bootstrap?   If so I assume some Bootstrap CSS is interfering with Galleria.

**Reply by kurtomerfaruk** [reply]
Bootstrap works with tab. However, for it to work, it must be in the active tab in the dialogue opening

**Reply by melloware** [reply]
Please provide an executable example using the [PrimeFaces Test](https://github.com/primefaces/primefaces-test) project.  It is the only way developers can debug your problem to help.

**Reply by kurtomerfaruk** [reply, marked as answer]
I solved the problem as follows. When the tab with Galleria is active, I re-render Galleria.

`
$(document).on('shown.bs.tab', 'a[data-toggle="tab"]', function (e) {
            var tab = $(e.target);
            var contentId = tab.attr("href");

            if (tab.parent().hasClass('active') &amp;&amp; contentId==="#resimlerEdit") {
                PF('galleriaWidget').render()
            }

        });
`
